core/templates/services/utils.service.ts

```diff
@@ -24,6 +24,17 @@ import { downgradeInjectable } from '@angular/upgrade/static';
   providedIn: 'root'
 })
 export class UtilsService {
+  // Determines if a variable is defined and not null.
```

Comment:
Can we add this under /** as we have started the comment below?

Reply:
Done!